src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
+  FaPlus, FaLinkedin, FaTwitter, FaInstagram,
+} from 'react-icons/fa';
+import './sidebar.css';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/home',
+      text: 'Events',
+    },
+    {
+      id: 2,
+      path: '/Myreservations',
+      text: 'Reservations',
+    },
+    {
+      id: 3,
+      path: '/Addevent',
+      text: 'Add event',
+    },
+    {
+      id: 4,
+      path: '/Addreservation',
+      text: 'Reserve',
+    },
+    {
+      id: 5,
+      path: '/',
+      text: 'sign out',
+    },
+  ];
```

Comment:
[OPTIONAL] You may consider storing this in another file and importing it and passing it to the component. It makes it easier to scale as your code base gets bigger and also makes for cleaner code.